app/experimenter/experiments/constants/nimbus.py

```diff
@@ -6,21 +6,95 @@
 from django.db import models
 
 
+class Channel(models.TextChoices):
+    NO_CHANNEL = ""
+    UNBRANDED = "default"
+    NIGHTLY = "nightly"
+    BETA = "beta"
+    RELEASE = "release"
+
+
+class BucketRandomizationUnit(models.TextChoices):
+    NORMANDY = "normandy_id"
+    NIMBUS = "nimbus_id"
+
+
+@dataclass
+class ApplicationConfig:
+    name: str
+    slug: str
+    app_name: str
+    channel_app_id: Dict[str, str]
+    default_app_id: str
+    randomization_unit: str
+
+
+APPLICATION_CONFIG_DESKTOP = ApplicationConfig(
+    name="Firefox Desktop",
+    slug="firefox-desktop",
+    app_name="firefox_desktop",
+    channel_app_id={
+        Channel.NIGHTLY: "firefox-desktop",
+        Channel.BETA: "firefox-desktop",
+        Channel.RELEASE: "firefox-desktop",
+    },
+    default_app_id="firefox-desktop",
+    randomization_unit=BucketRandomizationUnit.NORMANDY,
+)
+
+APPLICATION_CONFIG_FENIX = ApplicationConfig(
+    name="Firefox for Android (Fenix)",
+    slug="fenix",
+    app_name="fenix",
+    channel_app_id={
+        Channel.NIGHTLY: "org.mozilla.fenix",
+        Channel.BETA: "org.mozilla.firefox_beta",
+        Channel.RELEASE: "org.mozilla.firefox",
+    },
+    default_app_id="",
+    randomization_unit=BucketRandomizationUnit.NIMBUS,
+)
+
+APPLICATION_CONFIG_IOS = ApplicationConfig(
+    name="Firefox for iOS",
+    slug="ios",
+    app_name="firefox_ios",
+    channel_app_id={
+        Channel.NIGHTLY: "org.mozilla.ios.Fennec",
+        Channel.BETA: "org.mozilla.ios.FirefoxBeta",
+        Channel.RELEASE: "org.mozilla.ios.Firefox",
+    },
+    default_app_id="",
+    randomization_unit=BucketRandomizationUnit.NIMBUS,
+)
+
+
 @dataclass
 class NimbusTargetingConfig:
     name: str
     slug: str
     description: str
     targeting: str
     desktop_telemetry: str
+    application: ApplicationConfig
 
 
+TARGETING_NO_TARGETING = NimbusTargetingConfig(
```

Comment:
Adding this as a full-fledged targeting choice made some other code simpler. Shouldn't work the same otherwise

Reply:
Oh interesting.  Here we have `application=None` meaning it applies to all of them, I just asked @k88hudson and she suggested we may have targeting that applies to more than one application, so maybe this should be a list and in this example we put all three in?

Reply:
Switched over to using a list of application choice names for this